src/MigrationTools/Services/TelemetryClientAdapter.cs

```diff
@@ -19,6 +22,14 @@ public TelemetryClientAdapter(TelemetryClient telemetryClient)
                 telemetryClient.Context.Component.Version = System.Reflection.Assembly.GetEntryAssembly().GetName().Version.ToString();
             }
             _telemetryClient = telemetryClient;
+
+            elmahIoClient = ElmahioAPI.Create("7589821e832a4ae1a1170f8201def634", new ElmahIoOptions
+            {
+                Timeout = TimeSpan.FromSeconds(30),
+                UserAgent = "Azure-DevOps-Migration-Tools",
+            });
+            elmahIoClient.Messages.OnMessage += (sender, args) => args.Message.Version = System.Reflection.Assembly.GetEntryAssembly().GetName().Version.ToString();
```

Comment:
You could move assigning ´Version` to the `TrackException` method to simplify the code. You can set the `Version` property when you set all the other properties.